website/source/guides/web-ui/operating-a-job.html.md

```diff
@@ -0,0 +1,218 @@
+Sometimes tasks will have a memory leak, sometimes a node will have noisy neighbors, and sometimes
+we have no clue what's going so it's time to try turning it off and on again.
+
+For these times, Nomad allows for restarting and stopping individual allocations and tasks. When a
+task is restarted, Nomad will perform a local restart of the task. When an allocation is stopped,
+Nomad will mark the allocation as complete and perform a reschedule onto a different client node.
+
+Both of these features are also available in the Web UI.
+
+[![Allocation Stop and Restart][img-alloc-stop-restart]][img-alloc-stop-restart]
+
+## Forcing a Periodic Instance
+
+Periodic jobs are configured like a cron job. Sometimes, we want to micromanage the job instead of
+waiting for the period duration to elapse. Nomad calls this a
+[periodic force](/docs/commands/job/periodic-force.html) and it can be done from the Web UI on the
+Job Overview page for a periodic job.
+
+[![Periodic Force][img-periodic-force]][img-periodic-force]
+
+## Submitting a New Version of a Job
+
+From the Job Definition page, a job can be edited. After clicking the Edit button in the top-right
+corner of the code window, the job definition JSON becomes editable. The edits can then be planned
+and scheduled.
+
+[![Job Definition Edit][img-job-definition-edit]][img-job-definition-edit]
+
+~> Since each job within a namespace must have a unique name, it is possible to submit a new version
+of a job from the Run Job screen. Always review the plan output!
+
+## Monitoring a Deployment
+
+When a system or service job includes the [`update` stanza](/docs/job-specification/update.html), a
+deployment is created upon job submission. Job deployments can be monitored in realtime from the Web
+UI.
+
+The Web UI will show as new allocations become placed, tallying towards the expected total, and
+tally allocations as they becme healthy or unhealthy.
+
+Optionally, a job may use canary deployments to allow for additional health checks or manual testing
+before a full roll out. If a job uses canaries and is not configured to automatically promote the
+canary, the canary promotion operation can be done from the Job Overview page in the Web UI.
+
+[![Job Deployment with Canary Promotion][img-job-deployment-canary]][img-job-deployment-canary]
+
+## Stopping a Job
+
+Jobs can be stopped from the Job Overview page. Stopping a job will gracefully stop all allocations,
+marking them as complete, and freeing up resources in the cluster.
+
+[![Job Stop][img-job-stop]][img-job-stop]
+
+## Access Control
+
+Depending on the size of your team and the details of your Nomad deployment, you may wish to control
+which features different internal users have access to. This includes differentiation between
+submitting jobs, restarting allocations, and viewing potentially sensitive logs.
+
+Nomad has an access control list system for doing just that.
```

Comment:
Maybe take this line out: **Nomad has an access control list system for doing just that.**

Instead, just add a sentence like this at the end to the previous paragraph:

**You can enforce this with Nomad's access control list system.**